.github/CODEOWNERS

```
@@ -0,0 +1,20 @@
+# anything with no explicit code owners will be tagged to @core-devs
+*                       @napari/core-devs
+
+# submodules
+napari/_vispy/          @brisvag @melonora
+napari/_qt/             @Czaki @lucyleeow @DragaDoncila
+napari/_app_model/      @lucyleeow
+napari/benchmarks/      @Czaki
+napari/plugins/         @Czaki @DragaDoncila @lucyleeow
+napari/qt/              @Czaki
+napari/settings/        @Czaki @jni
+
+# specific layers
+napari/layers/image/    @Czaki @brisvag @andy-sweet
+napari/layers/labels/   @jni @Czaki @brisvag
+napari/layers/points/   @brisvag @kevinyamauchi @andy-sweet
```

```suggestion
napari/layers/points/   @brisvag @kevinyamauchi @andy-sweet @DragaDoncila 
```

I've worked a bit with `Points` and will be doing so even more with `Graph` layer, so I'm happy to be added here